src/odemis/driver/picoquant.py

```diff
@@ -782,7 +782,7 @@ def _acquire(self):
             logging.debug("Acquisition thread requested to terminate")
         except Exception:
             logging.exception("Failure in acquisition thread")
-        else:
+        else:  # TODO code unreachable?
```

Comment:
Indeed, looks like it's unreachable, as it's in a `while True`, without `break`. Not a big deal though, as it's just for logging. The "standard" way to exit this loop is actually the StopIteration exception.

Reply:
leave?

Reply:
remove or  leave this code?

Reply:
Let's just leave it as-is.